Tests/Common.lean

```diff
@@ -0,0 +1,34 @@
+/-
+Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author(s): Yan Peng
+-/
+import Arm.BitVec
+import Arm.FromMathlib
+
+def revflat (x : List (BitVec n)) : BitVec (n * x.reverse.length) := BitVec.flatten (List.reverse x)
```

Comment:
It would be cleaner to have the return type of this function be `BitVec (n * x.length)`. E.g.:

```
def revflat (x : List (BitVec n)) : BitVec (n * x.length) := 
  have h : x.reverse.length = x.length := by simp only [List.length_reverse]
  h ▸ BitVec.flatten (List.reverse x)
```